PROJECTS.md

```diff
@@ -0,0 +1,53 @@
+|    | name                                                         | project_type   | platforms          | language   |   ranking | holoscan_sdk.tested_versions   |
```

Comment:
Can we prettify this:
project_type -> project type
holoscan_sdk.tested_versions -> HSDK tested version

Reply:
Sure, I'll look into cleaning up the table as requested here and in other comments below